package.json

```diff
@@ -26,13 +26,13 @@
     "clean:tslint": "rimraf tslint-rules/*.js",
     "compile:tslint": "tsc -P tslint-rules",
     "compile:script": "tsc -P script/tsconfig.json",
-    "lint": "yarn lint:src && yarn lint:prettier",
-    "lint:fix": "yarn lint:src:fix && yarn lint:prettier --fix",
-    "lint:prettier": "ts-node script/prettier.ts",
+    "lint": "yarn prettier && yarn lint:src",
+    "lint:fix": "yarn prettier --write && yarn lint:src:fix",
```

Comment:
One problem with passing ` --write` into the `yarn prettier` that the previous script handled is that running `yarn lint:fix` when there is stuff to fix will return a non-zero exit code (because `--list-different` is still included?) leading to this bit of confusing output:

```shellsession
$  yarn lint:fix
yarn run v1.10.1
$ yarn prettier --write && yarn lint:src:fix
$ prettier --list-different "./**/*.{yaml,yml}" "./app/**/*.{scss,ts,tsx,js,jsx,html}" "./script/**/*.{ts,js}" --write
app/src/models/status.tsapp/src/models/status.ts
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I'd love for this to less terrifying, which was probably why I ended up going down the scripting path

Reply:
mmmm

Reply:
what if we did `yarn prettier --write; yarn lint:src:fix`? that way prettier can return whatever exit code it wants and we try to lint anyways?

Reply:
🤷‍♂️ it still bubbles up to the user

```shellsession
$ yarn lint:fix
yarn run v1.10.1
$ yarn prettier --write; yarn lint:src:fix
$ prettier --list-different "./**/*.{yaml,yml}" "./app/**/*.{scss,ts,tsx,js,jsx,html}" "./script/**/*.{ts,js}" --write
app/src/models/status.tsapp/src/models/status.ts
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
$ yarn tslint --fix && yarn eslint --fix
$ tslint -p . --fix
$ eslint --cache --rulesdir ./eslint-rules "./{script,eslint-rules,tslint-rules}/**/*.{j,t}s{,x}" "./app/{src,typings,test}/**/*.{j,t}s{,x}" "./changelog.json" --fix
```

Reply:
hmm i think this might be related to https://github.com/prettier/prettier/issues/4144. will dig in a little

Reply:
An alternative suggestion here might be to restore `script/prettier.ts` to keep the CLI experience friendly (for now) and pull in the new rules there, rather than hold up the rest of the `eslint` improvements.